dynatrace_health/README.md

```diff
@@ -0,0 +1,33 @@
+# Agent Check: Dynatrace_health
+
+## Overview
+
+This check monitors [Dynatrace][1] Events through the StackState Agent.
+
+## Setup
+
+### Installation
+
+The Dynatrace_health check is included in the [StackState Agent][2] package, so you do not
+need to install anything else on your server.
+
+### Configuration
+
+1. Edit the `dynatrace_health.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your dynatrace_health performance data.
+   See the [sample dynatrace_health.d/conf.yaml][2] for all available configuration options.
+
+2. Restart the Agent
+
+## Data Collected
+
+### Service Checks
+
+Dynatrace_topology includes a service check.
+
+### Events
+
+Dynatrace_health monitors Dynatrace events and creates health stream from them.
```

Comment:
..."and creates **a** health stream from them."

Reply:
How to access these events?

Reply:
In StackState Health states can be found on the right panel when a component is selected.